### ISSUE TITLE
citing article with two sources in a translation

### ISSUE THREAD

**Issue by JohnHeraty** [SpeciesFileGroup/taxonworks#1750]
**citing article with two sources in a translation**

For Entomological Reviews, the english translation is published in a different journal from the original russian article. In this example, there is no taxonomy, but if there were, then priority is to the russian article. Can one source be cited for two different journal sources.

**To Reproduce**

The source: Viggiani, G. (2015) Description of the final-instar larvae of the gall inducer wasps Aprostocetus monacoi Viggiani and Leptocybe invasa Fisher et La Salle (Hymenoptera, Chalcidoidea: Eulophidae). Entomological Review 95, 447–449. https://doi.org/10.1134/s0013873815040065

The problem:
<img width="805" alt="Screen Shot 2020-09-18 at 1 22 35 PM" src="https://user-images.githubusercontent.com/30123556/93641961-1af54d00-f9b2-11ea-97b7-6ab256dcb1cf.png">

This will be very rare, but there should be a note field or some other thing added to reference the two articles simultaneously.

**Environment (please identify where you experience this bug:**
- Production

**Comment by proceps** [comment]
I am running into the same problem. At the moment, I can save it only as a verbatim text in the Journal filed (not using serial). Of cause those could be saved as two separate sources, but for me this does not make much sense, it is the same source, but translated into different language. It would be nice to  have like a note field to add some additional information to the source, if there is no a relevant field.

**Comment by typophyllum** [comment]
Same problem in OSF, rather frequent, same journals. Different sources, somentimes cited both, more often the English version only, there both:

![grafik](https://user-images.githubusercontent.com/33210201/93841605-76516480-fc6a-11ea-9a60-d00cf1a402d3.png)

**Comment by proceps** [comment]
One down side of OriginRelationship, it is two directional. Two scenarios:
1. Bookchapter where the Book is Origin. In this case the Bookchapter is a primary source in focus, Origin Book may provide some supplementary information.
2. Translation in Entomological Review which has Russian version as an origin. In this case - Russin version is a primary source in focus, and Translation provides supplementary information.
In Second case, the relationship should be reversed.

**Comment by proceps** [comment]
Exactly, Translation is a new object, but we have to add information from translation to the old object (original version of publication), and use it as part of a citation (cached value).
When the species is described in original paper, and than translation was published (sometimes actually translation published first and servese as an origin for nomenclature), I want to see a single citation, not two separate citations in bibliography.
